vm/src/frame.rs

```diff
@@ -114,11 +114,16 @@ impl fmt::Debug for Frame {
             .map(|elem| format!("\n  > {:?}", elem))
             .collect::<Vec<_>>()
             .join("");
-        let local_str = "".to_string(); /* self.locals
-            .iter()
-            .map(|elem| format!("\n  {} = {}", elem.0, elem.1.borrow_mut().str()))
-            .collect::<Vec<_>>()
-            .join(""); */
+        let local_str = match self.locals.borrow().kind {
+            PyObjectKind::Scope { ref scope } => match scope.locals.borrow().kind {
+                PyObjectKind::Dict { ref elements } => format!(" {:?}", elements),
+                ref unexpected => format!(
+                    "locals unexpectedly not wrapping a dict! instead: {:?}",
```

Comment:
A panic is fine at this point I would say, since this is internally managed. If this is wrong, we are in really bad shape. It is not a python exception, but really a rust panic. Also consider `if let` for `match` statements with one arm only. It might be a good point to use here.